.github/dependabot.yml

```
@@ -0,0 +1,60 @@
+  - package-ecosystem: "docker"
+    directory: "/ccxt-service"
+    schedule:
+      interval: "weekly"
+      day: "monday"
+      time: "09:00"
+    open-pull-requests-limit: 3
+    commit-message:
+      prefix: "chore"
+      include: "scope"
+    reviewers:
+      - "irfandi"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)